package/lbcamden/all.js

```diff
@@ -1,1317 +1,43 @@
-    $button.querySelector('button').removeEventListener('click', $button.boundMenuItemClick, true);
-  });
-};
-
-LBCamdenHeader.prototype.handleMenuButtonClick = function () {
-  if (this.menuOpen === true) {
-    this.closeMenu(this.$navigationToggle, this.$navigationMenu);
-  } else {
-    this.openMenu(this.$navigationToggle, this.$navigationMenu);
-  }
-};
-
-LBCamdenHeader.prototype.handleSearchButtonClick = function () {
-  if (this.searchOpen === true) {
-    this.closeSearch(this.$searchToggle, this.$searchMenu);
-
-    if (this.mql.matches === true) {
-      this.$module.style.marginBottom = '0';
-      this.menuContentShift(0);
-    }
-  } else {
-    this.openSearch(this.$searchToggle, this.$searchMenu);
-
-    if (this.mql.matches === true) {
-      this.menuContentShift(this.$searchMenu.offsetHeight);
-    }
-  }
-};
-
-LBCamdenHeader.prototype.openMenu = function ($button, $target) {
-  this.menuOpen = true;
-  $button.classList.add('lbcamden-header__open-button');
-  $button.setAttribute('aria-expanded', !0);
-  $button.setAttribute('aria-label', 'Hide navigation menu');
-  $button.classList.add('lbcamden-header__open-button');
-  $target.removeAttribute('hidden');
-  if (this.$searchMenu != null) {
-    this.closeSearch(this.$searchToggle, this.$searchMenu);
-  }
-};
-
-LBCamdenHeader.prototype.closeMenu = function ($button, $target) {
-  this.menuOpen = false;
-  this.menuContentShift(0);
-  if (this.$navigationMenu != null) {
-    $button.classList.remove('lbcamden-header__open-button');
-    $button.setAttribute('aria-expanded', !1);
-    $button.setAttribute('aria-label', 'Show navigation menu');
-    $button.classList.remove('lbcamden-header__open-button');
-    $target.setAttribute('hidden', !0);
-  }
-};
-
-LBCamdenHeader.prototype.closeDesktopMenus = function () {
-  this.$module.querySelectorAll('.lbcamden-header__navigation-dropdown-menu').forEach(x => x.setAttribute('hidden', true));
-};
-
-LBCamdenHeader.prototype.openSearch = function ($button, $target) {
-  this.searchOpen = true;
-  $button.setAttribute('aria-expanded', !0);
-  $button.setAttribute('aria-label', 'Hide navigation menu');
-  this.$module.querySelectorAll('.lbcamden-header__open-button').forEach(x => x.classList.remove('lbcamden-header__open-button'));
-  $button.classList.add('lbcamden-header__open-button');
-  $target.style.top = this.$navContainer.offsetHeight + 'px';
-  $target.removeAttribute('hidden');
-  // document.getElementById('lbs-search__box').focus()
-  if (this.mql.matches !== true) {
-    this.closeMenu(this.$navigationToggle, this.$navigationMenu);
-  } else {
-    this.closeDesktopMenus();
-  }
-};
-
-LBCamdenHeader.prototype.closeSearch = function ($button, $target) {
-  this.searchOpen = false;
-  $button.setAttribute('aria-expanded', !1);
-  $button.setAttribute('aria-label', 'Show navigation menu');
-  $button.classList.remove('lbcamden-header__open-button');
-  $target.setAttribute('hidden', !0);
-};
+import './all.scss'
 
```

Comment:
consider whether we want this - it'll mean that any consumers of this will need to support bundling scss. another option would be to put it in index.html, which should also add it to the vite bundle